en_US/advanced/cluster.md

```diff
@@ -58,6 +58,51 @@ Cookies are used for interconnection authentication between Erlang nodes. A cook
 
 See <http://erlang.org/doc/reference_manual/distributed.html> for details.
 
+### Using TLS for backplane connections
+It is possible to enable TLS encryption for the backplane connections. It comes at the cost of increased CPU load, though.
+
+1. Create a root CA using `openssl` tool:
+   ```
+  # Create self-signed root CA:
+  openssl req -nodes -x509 -sha256 -days 1825 -newkey rsa:2048 -keyout rootCA.key -out rootCA.pem -subj "/O=LocalOrg/CN=LocalOrg-Root-CA"
+   ```
+2. Generate CA-signed certificates for the nodes using the rootCA.pem created at step 1:
+  ```
+  # Create a private key:
+  openssl genrsa -out domain.key 2048
+  # Create openssl extfile:
+  cat <<EOF > domain.ext
+  authorityKeyIdentifier=keyid,issuer
+  basicConstraints=CA:FALSE
+  subjectAltName = @alt_names
+  [alt_names]
+  DNS.1 = backplane
```

Comment:
Just for my own curiosity: can all the nodes share the same certificate with the same common name / subject alt name?

Reply:
Just saw your [other comment](https://github.com/emqx/emqx-docs/pull/1005#discussion_r797525559). :smiley_cat: